audit-logs/chart/ci/test-values.yaml

```diff
@@ -30,7 +30,7 @@ audit-logs-operator:
   kubeRBACProxy:
     enabled: false
 
-openTelemetry:
+auditLogs:
```

Comment:
Is this intentional? It will break the gh-action testing.

Reply:
Ah no, that should not be there. I renamed it back and forth a few times, must have missed this file.